docs/src/main/sphinx/connector/googlesheets.rst

```diff
@@ -98,6 +98,86 @@ may prevent the limit from being reached. Running queries on the ``information_s
 table without a schema and table name filter may lead to hitting the limit, as this requires
 fetching the sheet data for every table, unless it is already cached.
 
+Type mapping
+------------
+
+Because Trino and Google Sheets each support types that the other does not, this
+connector :ref:`modifies some types <type-mapping-overview>` when reading data.
```

Comment:
The connector supports only `TEXT` → `VARCHAR` mapping. Other types will be added in https://github.com/trinodb/trino/pull/5011

Reply:
I meant only one entry should exist at this time.